code/dynamic_programming/src/Array_Median/median.cpp

```diff
@@ -0,0 +1,26 @@
+#include<iostream>
+#include<algorithm>
+#include<vector>
+using namespace std;
+
+float median(int n, vector<int> arr)
```

Comment:
There is only one more improvement here that would be good to do, that is to pass only the `vector` as parameter and then get `n` with `arr.size()`. Something like:
```cpp
float median(vector<int> arr)
{
    int n = arr.size();
    ...
}
```